### PR TITLE
LRAC-10666 Test fix individual detail attributes

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-10666

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase [hunk 1]
```diff
@@ -520,19 +520,19 @@ definition {
...
 
 		task ("Search for property name") {
-			ACUtils.searchBar(searchItem = "classPK");
+			ACUtils.searchBar(searchItem = "createDate");
 
-			ACIndividualsDashboard.assertIndividualAttribute(key_individualAttribute = "classPK");
+			ACIndividualsDashboard.assertIndividualAttribute(key_individualAttribute = "createDate");
 
-			for (var individualAttribute : list "birthDate,contactId,createDate,email,familyName,givenName,jobTitle,modifiedDate") {
+			for (var individualAttribute : list "createDate,email,familyName,givenName,modifiedDate,timeZoneId,userId,uuid") {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase:527]: Hey @joshitagaki, I think you could make this change in these other cases too:

[Here 1](https://github.com/liferay-qa/liferay-portal/pull/1633/commits/76c140894feceeab2108463b1ef7e4cceaea1659#diff-e08bee4054c8cd607c5947261f89cdcc0fa1a688dbe5e3d2f96925669ca1bd55R505-R507)

[Here 2](https://github.com/liferay-qa/liferay-portal/pull/1633/commits/76c140894feceeab2108463b1ef7e4cceaea1659#diff-e08bee4054c8cd607c5947261f89cdcc0fa1a688dbe5e3d2f96925669ca1bd55R517-R519)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase [hunk 2]
```diff
@@ -544,7 +544,7 @@ definition {
 				ACIndividualsDashboard.assertIndividualAttribute(key_individualAttribute = "familyName");
 			}
 
-			for (var individualAttribute : list "birthDate,classPK,contactId,createDate,email,givenName,jobTitle,modifiedDate") {
+			for (var individualAttribute : list "createDate,email,familyName,givenName,modifiedDate,timeZoneId,userId,uuid") {
 				ACIndividualsDashboard.assertIndividualAttributeNotPresent(key_individualAttribute = "${individualAttribute}");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase:548]: Hey @joshitagaki, I didn't understand how **_familyName_** is in the list of attributes that are not present, but before you verified that the same attribute was present

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase (hunk 1 above)
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase:527]: Hey @joshitagaki, I didn't understand how _**createDate**_ is in the list of attributes that are not present, but before you verified that the same attribute was present

Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/KnownIndividualsSearch.testcase]: Thanks! I resolved the issues above

**Review by ViniLopes20 — APPROVED** [review]

**Comment by alexcandidos** [comment]
Hi @joshitagaki , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Review by alexcandidos — APPROVED** [review]

**Pull request closed without merging**